README.md

```diff
@@ -26,6 +26,8 @@ I summarize pretrained BERT models for Japanese by `word segmentation algorithm`
 | Tohoku University (b)      | MeCab (mecab-ipadic)                             | Character                                 | Sentencepiece (model_type=character)                                   |
 | NICT (a)                   | MeCab (mecab-jumandic)                           | WordPiece                                 | subword-nmt (BPE)                                                      |
 | NICT (b)                   | MeCab (mecab-jumandic)                           | ---                                       | ---                                                                    |
+| akirakubo (a)              | MeCab (unidic-cwj + unidic\_qkana)               | WordPiece                                 | subword-nmt (BPE)                                                      |
+| akirakubo (b)              | SudachiPy + MeCab (unidic\_qkana)                | WordPiece                                 | subword-nmt (BPE)                                                      |
```

Comment:
> SudachiPy + MeCab (unidic\_qkana)

Same question here. I think it implies the two analyzer configurations:

- (a) SudachiPy + SudachiDict_core (20191224)
- (b) MeCab + Unidic-qkana

I'd like to know how to combine the output of the analyzers in preprocessing. :eyes:

(https://github.com/akirakubo/bert-japanese-aozora#pre-tokenized-by-sudachipy-with-sudachidict_core-20191224-and-mecab-with-unidic-qkana_1603-2)

Reply:
Same as above.